config/routes.rb

```diff
@@ -1,24 +1,25 @@
 Rails.application.routes.draw do
-
-  devise_for :players
+  
+  devise_for :players, :controllers => { :omniauth_callbacks => "omniauth_callbacks" }
+  
   root 'games#index'
   resources :games
-
+  
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
-
+  
   # You can have the root of your site routed with "root"
   # root 'welcome#index'
-
+  
```

Comment:
FYI the more changes you make to the diff, the more complicated your life is during a conflict. With that in mind, I don't recommend stylistic changes such as this without an explicit reason.